ClassTypes/PowerShell/HyperV/check-hyperv-vm/README.md

```diff
@@ -0,0 +1,37 @@
+# Check my Hyper-V VMs
+
+[Test-HypervVmConfig](./Test-HypervVmConfig.ps1) will verify you setup follows best practices for nested virtualization.  
```

Comment:
Maybe mention it's best practices for nested virtualization on Azure Lab Services?  Also - it might be good to specify there that this is the best script to run before exporting an image to the gallery (if that's the plan for the template)